model/EligibilityService.php

```diff
@@ -146,11 +151,20 @@ public function getEligibleTestTakers(Resource $testCenter, Resource $delivery)
      * @return boolean
      */
     public function setEligibleTestTakers(Resource $testCenter, Resource $delivery, $testTakerIds) {
+        /** @var \core_kernel_classes_Resource $eligibility */
         $eligibility = $this->getEligibility($testCenter, $delivery);
         if (is_null($eligibility)) {
             throw new IneligibileException('Delivery '.$delivery->getUri().' ineligible to test center '.$testCenter->getUri());
         }
-        return $eligibility->editPropertyValues(new Property(self::PROPERTY_TESTTAKER_URI), $testTakerIds);
+
+        $previousTestTakerCollection = $eligibility->getPropertyValues(new Property(self::PROPERTY_TESTTAKER_URI));
```

Comment:
This change breaks unit test of `EligibilityService`.
Call of `getPropertyValues()` method is not expected for `$eligibility` resource mock:
https://github.com/oat-sa/extension-tao-proctoring/blob/develop/test/EligibilityServiceTest.php#L283